src/pipeline-status-slack.py

```diff
@@ -116,8 +116,10 @@ def lambda_handler(event, context):
     footer = ""
     if manually_triggered:
         footer = "Triggered by AWS Console"
-    elif all(key in execution_input for key in ["git_user", "git_repo", "git_branch"]):
-        footer = f"Triggered by {execution_input['git_user']} @ {execution_input['git_repo']} ({execution_input['git_branch']})"
+    elif all(execution_input.get(key, None) for key in ["git_user", "git_repo", "git_branch"]):
+        footer = f"Triggered by {execution_input['git_user']} from repository {execution_input['git_repo']} [{execution_input['git_branch']}]"
+    elif all(execution_input.get(key, None) for key in ["git_repo", "git_branch"]):
+        footer = f"Triggered from repository {execution_input['git_repo']} [{execution_input['git_branch']}]"
```

Comment:
Burde det forøvrig ikke kanskje være en `else` for en potensielt ukjent trigger?

Reply:
Bra innspill! Vi styrer alle disse feltene fra upstream, så det er i hovedsak kun disse tre tilfellene vi kommer til å treffe på her. Per nå er fallbacken bare en tom streng (`footer = ""`) hvis ingen matcher. Mulig denne kan endres til noe à la `Unable to determine triggering details from execution input`?

Reply:
... og i tillegg er det for å være bakoverkompatibel. Disse feltene er ikke inkludert i eldre versjoner av pipeline-oppsettet. I så fall tror jeg det er bedre å defaulte til en tom streng fremfor noe som ligner på en feilmelding